pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -289,6 +289,9 @@ type MetadataServerSpec struct {
 	// The number of metadata servers that are active. The remaining servers in the cluster will be in standby mode.
 	ActiveCount int32 `json:"activeCount"`
 
+	// The number of standby metadata servers that are active.
+	ActiveStandbyCount int32 `json:"activestandbycount"`
```

Comment:
How does this option deal with `ActiveStandby` aren't conflicting with each other?

Reply:
When testing, I set `activestandbycount` to `activestandbycount: 1` in `cephfilesystem`. One new MDS pod was coming up. So, there were a total 3 MDS pods, 1 active and 2 active standby.